packages/@aws-cdk/aws-codepipeline/test/pipeline.test.ts

```diff
@@ -336,6 +340,127 @@ describe('', () => {
           'arn:${AWS::Partition}:iam::123456789012:role/cdk-hnb659fds-cfn-exec-role-123456789012-us-west-2');
 
 
+      });
+      test('generates the same support stack containing the replication Bucket without the need to bootstrap in that environment for multiple pipelines', () => {
+        const app = new cdk.App({
+          treeMetadata: false, // we can't set the context otherwise, because App will have a child
+        });
+        app.node.setContext(cxapi.NEW_STYLE_STACK_SYNTHESIS_CONTEXT, true);
+
+        class PipelineStack extends cdk.Stack {
+          public constructor(scope: Construct, id: string, props: cdk.StackProps ) {
+            super(scope, id, props);
+            const sourceOutput = new codepipeline.Artifact();
+            new codepipeline.Pipeline(this, 'Pipeline', {
+              stages: [
+                {
+                  stageName: 'Source',
+                  actions: [new FakeSourceAction({
+                    actionName: 'Source',
+                    output: sourceOutput,
+                  })],
+                },
+                {
+                  stageName: 'Build',
+                  actions: [new FakeBuildAction({
+                    actionName: 'Build',
+                    input: sourceOutput,
+                    region: 'eu-south-1',
```

Comment:
Can you actually add two actions in this region? I think that will test that we create a single support stack for both of them.